tvb_build/hbp_openshift_deployer/tvb_deployer.yml

```diff
@@ -292,6 +292,9 @@ objects:
                   cpu: '3'
                 requests:
                   cpu: '1'
+              env:
+                - name: APP_ENCRYPTION_HANDLER
+                  value: 'true'
```

Comment:
@popaula937 We recently found this flag from missing from the YML by reading through the code (on what is expected). Are you aware of any reason why this should not be added here ?

Reply:
No reason against it, most probably was forgotten while working on this setup.